drivers/scheduler/dcos/errors.go

```diff
@@ -53,3 +53,39 @@ type ErrFailedToValidateAppDestroy struct {
 func (e *ErrFailedToValidateAppDestroy) Error() string {
 	return fmt.Sprintf("Failed to validate destroy of app: %v due to err: %v", e.App.Key, e.Cause)
 }
+
+// ErrFailedToValidateVolume error type for failing to validate a volume
```

Comment:
Why are these errors separate for the schedulers? There seems to be nothing scheduler specific and both the schedulers seem to have the same errors re-defined

Reply:
yes there is no need to have them separate.
There were some errors which were PVC or Storage Class specific. I made it generic and extracted in scheduler/errors